bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/handler/TelldusDevicesHandler.java

```diff
@@ -151,24 +151,26 @@ private void refreshDevice(Device dev) {
     public void initialize() {
         Configuration config = getConfig();
         logger.debug("Initialize TelldusDeviceHandler {}. class {}", config, config.getClass());
-        final Object configDeviceId = config.get(TellstickBindingConstants.DEVICE_ID);
-        if (configDeviceId != null) {
-            deviceId = configDeviceId.toString();
-        } else {
-            logger.debug("Initialized TellStick device missing serialNumber configuration... troubles ahead");
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR);
-        }
         final Boolean isADimmer = (Boolean) config.get(TellstickBindingConstants.DEVICE_ISDIMMER);
         if (isADimmer != null) {
-            this.isDimmer = isADimmer;
+            isDimmer = isADimmer;
         }
         final BigDecimal repeatCount = (BigDecimal) config.get(TellstickBindingConstants.DEVICE_RESEND_COUNT);
         if (repeatCount != null) {
             resend = repeatCount.intValue();
         }
-        Bridge bridge = getBridge();
-        if (bridge != null) {
-            bridgeStatusChanged(bridge.getStatusInfo());
+        final Object configDeviceId = config.get(TellstickBindingConstants.DEVICE_ID);
+        if (configDeviceId != null) {
+            deviceId = configDeviceId.toString();
+            Bridge bridge = getBridge();
+            if (bridge != null) {
+                bridgeStatusChanged(bridge.getStatusInfo());
+            } else {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_UNINITIALIZED);
```

Comment:
Is this needed/correct? If bridge is uninitialized, so are child handlers? If thing is configured without a bridge, this is a configuration error.

Reply:
Changed to a CONFIGURATION_ERROR